python/apollo_openapi/model/open_env_namespace_role_user_dto.py

```
@@ -0,0 +1,170 @@
+
+                class MetaOapg:
+
+                    @staticmethod
+                    def items() -> typing.Type['OpenUserInfoDTO']:
+                        return OpenUserInfoDTO
+
+                def __new__(
+                    cls,
+                    _arg: typing.Union[typing.Tuple['OpenUserInfoDTO'], typing.List['OpenUserInfoDTO']],
+                    _configuration: typing.Optional[schemas.Configuration] = None,
+                ) -> 'modifyRoleUsers':
```

Ruff (0.14.8)

Undefined name `modifyRoleUsers`

(F821)